import.py

```diff
@@ -300,6 +349,19 @@ def findFieldType(sf, fieldName):
         exit()
 
 
+# See https://docs.djangoproject.com/en/2.2/ref/contrib/gis/gdal/#django.contrib.gis.gdal.GDALBand.datatype for the possible values
```

Comment:
I wrote this function at the start of the process, when it wasn't yet clear that FloatField()s could cause rounding/comparison problems.  I should either make it reject float fields, or warn the user that float values will be rounded to ints, or add the other code elsewhere that would attempt to handle real number values.

Reply:
This seems worth talking to Carson about, as in - how often does she find floats in rasters like this, and what should be done about it?